src/components/TodoList.js

```diff
@@ -45,13 +45,21 @@ function TodoList() {
             return todo;
         })
         setTodos(updatedTodos);
+        console.log(todos)
     };
+    const deleteAll = id =>{
+        console.log(id)
+        const removeArr = [...todos].filter(todo => console.log(todo));
```

Comment:
Looks like this filter doesn't do anything. Please review the logic.

Reply:
Made the changes